mobile.css

```diff
@@ -277,6 +277,143 @@ button,
   justify-content: center;
 }
 
+#modal-container {
+  display: block;
+  position: fixed;
+  top: 0;
+  left: -100%;
+  background-color: #191765cc;
+  backdrop-filter: blur(3px);
+  width: 100vw;
+  height: 100vh;
+  opacity: 80%;
+  transition: all 1s ease-out;
+}
+
+#modal-container.show {
+  display: block;
+  opacity: 100%;
+  left: 0;
+  pointer-events: auto;
+  overflow-y: auto;
+}
+
+#modal {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin: 2em 1.5em 1em;
+  width: auto;
+  height: fit-content;
+  background-color: rgba(255, 255, 255, 0.2);
+  mix-blend-mode: screen;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  border: 1px solid white;
+}
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** I think it would be better if you add the background icons(or image) both on the **mobile** and **desktop pop-up windows** to make your project stick to the Figma design.

- [ **OPTIONAL** ]Good job by making the changes required by the previous reviewer. I think it would be better if you remove the **scrolling** on the **body content** when you are watching the popup window so the user can not interact with body content.